### PR TITLE
OCM Insights tab behaviour

### LINKED ISSUE #11: Prepare new behavioral specification: Insights tab for cluster w/o any recommendations available
Prepare new behavioral specification: Insights tab for cluster w/o any recommendations available

### LINKED ISSUE #12: Prepare new behavioral specification: Insights tab for cluster with no issues
Prepare new behavioral specification: Insights tab for cluster with no issues

### LINKED ISSUE #13: Prepare new behavioral specification: Insights tab for cluster with at least one issue
Prepare new behavioral specification: Insights tab for cluster with at least one issue

### PR DESCRIPTION
**Pull request by tisnik**
Fixed #11 
Fixed #12 
Fixed #13

### CODE REVIEW

File: features/OCM_insights_tab.feature [hunk 1]
```diff
@@ -0,0 +1,90 @@
+Feature: Insights tab on OCM UI
+
+  Scenario: Insights tab for cluster without any issue
+    Given console.redhat.com is accessible
+      And user U1 is already loged in into console.redhat.com
```
Comment (epapbak) [inline features/OCM_insights_tab.feature:5]: logged

File: features/OCM_insights_tab.feature [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      And user U1 switch to "OpenShift" page in console.redhat.com (OCM UI)
+      And user U1 is part of account (organization) at least one cluster C1
```
Comment (epapbak) [inline features/OCM_insights_tab.feature:7]: "account with"?

File: features/OCM_insights_tab.feature [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+     When user U1 click on cluster name
+     Then a page with detailed information about the selected cluster should be displayed
+      And at least four tab headers should be displayed on that page
+          | Tab name         |
+          | Overview         |
+          | Monitoring       |
+          | Insights Advisor |
+          | Support          |
+     When user U1 clicks on tab named "Insights Advisor"
+     Then new page should be displayed saying "Your cluster passed all recommendations"
```
Comment (epapbak) [inline features/OCM_insights_tab.feature:24]: This is what I see

image

File: features/OCM_insights_tab.feature [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      And button with text "What is insights?" is displayed on this page
```
Comment (epapbak) [inline features/OCM_insights_tab.feature:25]: see previous comment

File: features/OCM_insights_tab.feature (hunk 1 above)
Comment (tisnik) [reply features/OCM_insights_tab.feature:5]: nice catch, will fix it

File: features/OCM_insights_tab.feature [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+     Then new page should be displayed saying "1 potential issue identified"
+      And the page should contains several sections
+      And "Recommendations by category section" should display the following data
+          | Attribute            | Value |
+          | Fault Tolerance      | 0     |
+          | Performance          | 0     |
+          | Security             | 1     |
+          | Service Availability | 0     |
+      And exactly one rule name should be displayed in the third section
+     When user expands the rule by clicking on ">" icon
```
Comment (radekvokal) [inline features/OCM_insights_tab.feature:86]: or by clicking on name of the recommendations 

Also please s/rule/recommendation. Rule is the condition that we run internally, recommendation is the condition + content.

**Pull request merged**